script.js

```diff
@@ -10,34 +13,45 @@ function computerSelection() {
     return randomComputerSelection;
 }
 
-function playRound() {
-    //let playerInput =  prompt("Choose Rock, Paper, or Scissors!");
-    //let playerSelection = playerInput.toLowerCase();
-    //let computersMove = computerSelection();
-
-    if (
+//plays a single round and produces round winner
+function playRound(playerSelection) {
```

Comment:
```
Was thinking something more like this

function playRound(playerSelection) {
    let computersMove = computerSelection();
    let roundWinnerMessage = document.getElementById("winnerMessage");
    let winnerMessage = '';

    if (
        (playerSelection === choiceRock && computersMove === choiceScissors) ||
        (playerSelection === choiceScissors && computersMove === choicePaper) ||
        (playerSelection === choicePaper && computersMove === choiceRock)) {
        winnerMessage = `Player 1 wins! ${playerSelection.toUpperCase()} beats ${computersMove.toUpperCase()} `;
        document.querySelector(".p1-score").textContent = (playerOneScore++) + 1;
    } else if (
        (playerSelection === choiceRock && computersMove === choicePaper) ||
        (playerSelection === choiceScissors && computersMove === choiceRock) ||
        (playerSelection === choicePaper && computersMove === choiceScissors)) {
        winnerMessage =  `Computer wins! ${computersMove.toUpperCase()} beats ${playerSelection.toUpperCase()}`;
        document.querySelector(".p2-score").textContent = (playerTwoScore++) + 1;
    } else if (playerSelection === computersMove) {
        roundWinnerMessage.innerHTML = drawGameMessage;
   }
    winnerMessage  = `Draw Game! ${playerSelection.toUpperCase()} vs ${computersMove.toUpperCase()} `;;
    displayFinalMessage();
}
```

Also these the conditions on these if statements are looking pretty ugly and hard to read. It might be nice to make a function that returns what player won and use that in the if conditions